### PR TITLE
[MRG] Game loader

### LINKED ISSUE #3: Structure of calling python code from a python process using subprocess
I'd be interested in knowing what this is aiming to achieve:

https://github.com/JamesDevJim/game-zulu/blob/4b46243499803992d0ef07bb856fb49bb8f72240/game_changer.py#L20-L30

We already have a Python process so using Python to spawn a new Python process to run Python code seems to be much better done via other means, there's better ways of running Python code from within Python processes. However the specifics of what I'd suggest would depend a bit on what this is trying to achieve (imports vs exec vs other approaches).

### PR DESCRIPTION
**Pull request by shuttle1987**
Work on enabling a more effective manner of loading the sequence of games.

Closes #3

### CODE REVIEW

File: games/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Sentinel for the first game in the series
+FIRST_GAME = object()
+
+# Sentinel object for the last game in the series
+LAST_GAME = object
```
Comment (JamesDevJim) [inline games/__init__.py]: Should there be parentheses after 'object'?

Comment (JamesDevJim) [inline games/__init__.py]: Should there be parentheses after 'object'?

Comment (JamesDevJim) [inline games/__init__.py]: Should there be parentheses after 'object'?

**Review by JamesDevJim — COMMENTED** [review]
FWIW, I tried running game_entry.py and I got the following message.

image

File: games/__init__.py (hunk 1 above)
Comment (shuttle1987) [reply games/__init__.py]: Yes, good catch

Comment (shuttle1987) [reply games/__init__.py]: Fixed in adad2ae

**Comment by shuttle1987** [comment]
Give this a test and see if it works with the real hardware. There's a possibility that the initialization of the device connections to the embedded devices is subtly different here on the transitions between games, so let me know if that's been broken and I can help you fix that.

**Comment by JamesDevJim** [comment]
During a game zulu win, game stops. 

image

**Comment by shuttle1987** [comment]
> During a game zulu win, game stops.
> 

This specific bug is dealt with in e4d1e27

**Pull request merged**